src/drivers/frsky_telemetry/frsky_data.c

```diff
@@ -218,11 +218,11 @@ void frsky_send_frame1(int uart)
 {
 	/* send formatted frame */
 	frsky_send_data(uart, FRSKY_ID_ACCEL_X,
-			roundf(sensor_combined->accelerometer_m_s2[0] * 1000.0f));
+			roundf(sensor_combined->accelerometer_m_s2[0] * 1000.0f * (1 / 9.81f)));
 	frsky_send_data(uart, FRSKY_ID_ACCEL_Y,
-			roundf(sensor_combined->accelerometer_m_s2[1] * 1000.0f));
+			roundf(sensor_combined->accelerometer_m_s2[1] * 1000.0f * 1000.0f * (1 / 9.81f)));
```

Comment:
According to the OpenTX source [here](https://github.com/opentx/opentx/blob/dbd8abbfe8343b5d7f542304d47e232140307b95/radio/src/telemetry/frsky_d_arm.cpp#L122) and [here](https://github.com/opentx/opentx/blob/65d7fd0a290b48af2a9a8d9f9be076905a2c8e4a/radio/src/telemetry/frsky_sport.cpp#L74), these values are expected to be in g.
Since accelerometer_m_s2 is in m/s^2 I'm multiplying by 1/9.81m/s^2 to get the value in g.